lib/rototiller/task/collections/param_collection.rb

```diff
@@ -52,6 +59,12 @@ def filter_contents(filters={})
         filtered
       end
 
+      #@private
+      def allowed_class
+        EnvVar
```

Comment:
Should we include a #TODO noting that this is required for the tests and should be removed once fixed?